classes/requests/helpers/class-dintero-checkout-order.php

```diff
@@ -35,7 +43,7 @@ public function __construct( $order ) {
 	 */
 	public function get_order_total() {
 		$order_total = self::format_number( $this->order->get_total() );
-		if ( $this->order instanceof WC_Order ) {
+		if ( ! $this->is_refund ) {
```

Comment:
Issue: This check is not the same as `$this->order instanceof WC_Order`, and should not be used as a replacement. The old check made sure the order was a valid order, not that it was not a refund. This only checks that its not a refund order, and nothing else.

Since `WC_Order_Refund` extends `WC_Abstract_Order`, having the check for `WC_Order` should be fine here still.

Reply:
Good point.